### ISSUE TITLE
As a user I want to be able to play a pack from a discord command

### ISSUE THREAD

**Issue by mikalandrewg** [BallChasingCoach/Bot#33]
This one will require a heavy lift on the web side before we do anything with the bot. Essentially implement a !play command that returns a single pack with a play button. That play button will route to a site page which will automatically open the pack if they have the plugin running, or give them an error if we can't load the pack and explain why.

**Comment by rspraymond** [comment]
JS triggers the play correct? We probably just need a query parameter that you can pass on the pack page so that when you click the button it hits opens the website page which then autoloads the plugin or yeah throws up a message promoting the plugin.

**Comment by rspraymond** [comment]
I don't think this would be a "heavy lift" on the web side. Though we don't really size our features so hard to compare against something else lol

**Comment by mikalandrewg** [comment]
Good point. We just have to update the "Play" action to throw a generic error if it gets called and the websocket isn't connected to the plug-in. At that point yeah it's just passing a parameter into the pack page with a long enough timeout to connect.
